src/interract.js

```diff
@@ -0,0 +1,19 @@
+// Checkbox function
+export function checkvalue(list) {
+  document.addEventListener('change', (e) => {
+    if (e.target.classList.contains('to-do')) {
+      const index = e.target.dataset.listIndex - 1;
+      e.target.parentNode.children[1].classList.toggle('checked');
+
+      list.checked(index);
+    }
+  });
```

Comment:
![image](https://github.com/sylvester-wamaya/to-do-list/assets/93039359/2385e1cb-cb48-480b-9dbe-9c260fcd68f5)

- [ ] So I have done the below steps and faced this error which you can see in the image above ⬆️
 
1. I created 6 todos
2. Then I removed 1 to 4 and 6th todo
3. Then there was only one todo of index 5
4. When I tried to check and uncheck that 5th todo then I saw this error 😨

How you can fix it 👍

- [ ] It would be nice if you reset all the indexes of todos after clearing the completed todos then this error will be resolved 😊